config/webpack.config.dev.js

```diff
@@ -8,7 +8,6 @@ const CspHtmlWebpackPlugin = require('csp-html-webpack-plugin');
 const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
-const WatchMissingNodeModulesPlugin = require('react-dev-utils/WatchMissingNodeModulesPlugin');
```

Comment:
No longer a part of `react-dev-utils`, not really that important given it's just dev config anyways